lib/Net/DRI/DRD.pm

```diff
@@ -1454,7 +1454,7 @@ sub _build_price_query
  }
  my $fee_version = 0+($ndr->protocol()->{brown_fee_version} // $ndr->protocol()->{fee_version} // 0);
  # CentralNic::Fee (<=011) this extension is used in addition to premium_domain above, hense to elsif.
- if ((grep $_ eq 'Net::DRI::Protocol::EPP::Extensions::CentralNic::Fee', @{$ndr->protocol()->{loaded_modules}}) || $fee_version <= 0.11)
+ if ((grep $_ eq 'Net::DRI::Protocol::EPP::Extensions::CentralNic::Fee', @{$ndr->protocol()->{loaded_modules}}) || ( $fee_version <= 0.11 && $fee_version > 0 )  )
```

Comment:
What does this additional condition do? Is fee_version somehow less that 0 sometimes?

Reply:
added by me at the time when testing incorrect domain check price command :p